ScreenControl_Unity/Assets/ScreenControl/Core/Scripts/Configuration/SettingsUI/AutoConfig_Test.cs

```diff
@@ -121,15 +121,15 @@ public void RandomlyPlaceLeap()
 
         public void PlaceLeapTop()
         {
-            ConfigurationSetupController.selectedMountType = MountingType.OVERHEAD;
+            ConfigurationSetupController.selectedMountType = MountingType.ABOVE_FACING_SCREEN;
```

Comment:
Let's get rid of this seeing as it's not used and isn't actually part of the test plan